aws/resource_aws_ses_configuration_set.go

```diff
@@ -78,29 +79,27 @@ func resourceAwsSesConfigurationSetDelete(d *schema.ResourceData, meta interface
 	return err
 }
 
-func findConfigurationSet(name string, token *string, meta interface{}) (bool, error) {
+func findConfigurationSet(name string, meta interface{}) (bool, error) {
```

Comment:
alternatively this method's logic could be moved to the `Read` method @ L54 since this method is only called by the `Read` method atm and as an added bonus, the response to `DescribeConfigurationSet` can then be used to set the `name` attribute @ L66; in doing so, the error received from the `DescribeConfigurationSet` can also be directly evaluated (as described in the comment below) and returned as needed

Reply:
Hey @anGie44 - thanks for the great feedback, really appreciate the time and effort. I've addressed all of these... I did leave `findConfigurationSet` in place but I can look at moving that too

Reply:
sure thing @carthewd! thanks for addressing the comments :) re: this function, i think it would be a nice-to-have but definitely up to you as it predates this PR..imo moving the logic to the`Read`method of the resource would improve readability, especially with the error handling in place, and as a rule of thumb we should use the API response values to set schema values like `name`

Reply:
also I see github is reporting merge conflicts so would you mind rebasing when you have a chance?

Reply:
Thanks again @anGie44 - that does make sense, I've refactored it and it does seem a lot cleaner to me. Also fixed up the merge conflicts